<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>711</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1087</width>
    <height>711</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1087</width>
    <height>711</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="log_view">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>1021</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="theMap" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>529</width>
      <height>0</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="insert_button">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>211</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加新的旅客行程</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>351</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>计时器</string>
    </property>
    <widget class="QPushButton" name="start">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开始计时</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>停止计时</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restart">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>340</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>计时器重置
并刷新系统</string>
     </property>
    </widget>
    <widget class="QLabel" name="hourlabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>小时</string>
     </property>
    </widget>
    <widget class="QLabel" name="day_label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>天</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="hour_lcdNumber">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="day_lcdNumber">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>log_view</zorder>
   <zorder>insert_button</zorder>
   <zorder>groupBox</zorder>
   <zorder>theMap</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
